lib/kubernetes-deploy/kubernetes_resource/stateful_set.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+require 'kubernetes-deploy/kubernetes_resource/pod_set_base'
+module KubernetesDeploy
+  class StatefulSet < PodSetBase
+    TIMEOUT = 5.minutes
```

Comment:
Do you think this is long enough for statefulsets? Since they roll out one by one, I'd tend to think they'd need something more generous. Maybe check with DataDelivery?

Reply:
Good point. I'll check with them

Reply:
What did they say?